backend/canvas_app_explorer/views.py

```diff
@@ -12,18 +12,60 @@
 from backend.canvas_app_explorer.canvas_lti_manager.django_factory import DjangoCourseLtiManagerFactory
 from backend.canvas_app_explorer.canvas_lti_manager.exception import CanvasHTTPError
 
+from rest_framework_tracking.models import APIRequestLog
+from rest_framework_tracking.mixins import LoggingMixin
+from django.utils import timezone
+from django.db import transaction
+
+
+
 logger = logging.getLogger(__name__)
 
 MANAGER_FACTORY = DjangoCourseLtiManagerFactory(f'https://{settings.CANVAS_OAUTH_CANVAS_DOMAIN}')
 
-
-class LTIToolViewSet(viewsets.ViewSet):
+class LTIToolViewSet(LoggingMixin, viewsets.ViewSet):
     """
     API endpoint that lists LTI tools available in the course context, and allows for enabling/disabling navigation.
     """
     authentication_classes = [authentication.SessionAuthentication]
     permission_classes = [permissions.IsAuthenticated]
 
+    # Customize what gets logged
+    def handle_log(self):
+        """Hook to customize the log entry"""
+        logger.info("Logging API request...")
+
+        # Extract extra_data separately since it's handled differently
+        extra_data = {
+            'custom_field': 'custom_value',
```

Comment:
Is this necessary and/or does it do anything? It seems like setting these to static common names isn't useful? Are these needed by the library?